### PR TITLE
Add Transactions list column with Subscription link

### LINKED ISSUE #866: Add Subscriptions Column to Transactions List View (1/3)
paJDYF-O8-p2

Related: #867 #868

### PR DESCRIPTION
**Pull request by dechov**
Fixes #866

#### Changes proposed in this Pull Request

If WooCommerce Subscriptions is active, add links to any Subscription(s) associated with the relevant Order from a new cell in each row of the Transactions list.

subscription-link

(Note: most orders would likely have 0 or 1 subscriptions – multiple subscriptions would result from checking out with 2 or more subscriptions that have different renewal schedules.)

If there is no related subscription, then the cell is left empty.

#### Testing instructions

Purchase subscription(s) via WooCommerce Payments, and verify:
- Admin links to the subscription(s) appear in the Transactions list
- Payments that don't involve subscriptions should show nothing in that column
- If the subscriptions extension is not active, the column should not appear

-------------------

- [ ] Added changelog entry (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

### CODE REVIEW

File: includes/admin/class-wc-payments-admin.php [hunk 1]
```diff
@@ -181,11 +181,11 @@ public function register_payments_scripts() {
 			'WCPAY_DASH_APP',
 			'wcpaySettings',
 			[
-				'connectUrl'         => WC_Payments_Account::get_connect_url(),
```
Comment (DanReyLop) [inline includes/admin/class-wc-payments-admin.php:184]: `wcpaySettings.connectUrl` is used [here](https://github.com/Automattic/woocommerce-payments/blob/c7f9d58e2d20fc5fb10d3fa5d205f698bcf3b8c6/client/connect-account-page/index.js#L75).

File: client/transactions/list/index.js [hunk 2]
```diff
@@ -124,9 +138,12 @@ export const TransactionsList = ( props ) => {
 		isLoading: isSummaryLoading,
 	} = useTransactionsSummary( getQuery(), props.depositId );
 
-	const columnsToDisplay = props.depositId
-		? columns
-		: [ ...columns, depositColumn ];
+	const columnsToDisplay = useMemo( () => {
```
Comment (DanReyLop) [inline client/transactions/list/index.js:141]: Nice improvement!

**Review by DanReyLop — APPROVED** [review]
Works great! Add back the `connectUrl` key to the `wcpaySettings` array and after that you can :shipit:

File: includes/admin/class-wc-payments-admin.php (hunk 1 above)
Comment (dechov) [reply includes/admin/class-wc-payments-admin.php:184]: Thanks – no idea how that happened! Fixed in https://github.com/Automattic/woocommerce-payments/pull/925/commits/cbe11d15b29051d0ab8187f174c1841da0daa89c

**Pull request merged**